Grammar/python.gram

```diff
@@ -578,6 +578,8 @@ kwarg_or_double_starred[KeywordOrStarred*]:
 # NOTE: star_targets may contain *bitwise_or, targets may not.
 star_targets[expr_ty]:
     | a=star_target !',' { a }
+    | '(' a=starred_expression ')' {
+        RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "starred assignment target must be in a list or tuple") }
```

Comment:
If we prefer to also have:

```
*a = 1,2,3
```

raise `starred assignment target cannot be used with parentheses around`, then we can remove this line.